app/policies/linked_profile_policy.rb

```diff
@@ -0,0 +1,11 @@
+class LinkedProfilePolicy < ApplicationPolicy
+  def show?
+    record.user == user || user.has_role?(:admin)
```

Comment:
No, `alias_method from, to` means "create a method with `from` name that just defers to `to`"

So that basically creates the following method:

```ruby
def create?(*args)
  return is_admin?(*args)
end
```

It does not affect scoping, it's not magic.